.github/workflows/bot-ethereum-phosphore.yml

```diff
@@ -0,0 +1,54 @@
+name: "[Bot] 'Ethereum on Phosphore'"
```

Comment:
please remove this file, we stopped doing these kind of config files. you can directly use bot-phosphore.yml to trigger flows.